deep_learning_index.html

```diff
@@ -0,0 +1,15 @@
+---
+title: Deep Learning with PyTorch
+permalink: deep-learning-with-pytorch
+deep-learning-index: true
+---
+
+<!DOCTYPE html>
+<html lang="en">
+
+  {% include head.html %}
+
+  {% include deep_learning_content.html %}
+
+</html>
+
```

Comment:
You can take this code and move it into `deep_learning_content.html`. You can do the same thing with  `deep_learning_thank_you.html`. Instead of having `deep_learning_content.html` be an includes you can make it a layout. Then you can reference the layout in this file and in `deep_learning_thank_you.html`. Also, instead of using snake case for the file names (`deep_learning_index.html` and `deep_learning_thank_you.html`), you should use hyphens to separate the words, like `deep-learning-index.html`. That's how the rest of the pages are set up. Snake case is fine for the includes files, though. You can also put both of the files in the same folder.